Sources/Commands/Workspace.swift

```diff
@@ -287,6 +323,39 @@ public class Workspace {
         try removeFileTree(dataPath)
     }
 
+    /// Find the managed dependencies for the given package name.
+    func findDependency(forURL url: String) throws -> ManagedDependency? {
```

Comment:
or do you think that is fine? It probably only needs to be called once right now..